src/api/authentication/authenticateAdminRoutes.js

```diff
@@ -1,9 +1,32 @@
+import coBody from 'co-body';
+import config from 'config';
+import jwt from 'jsonwebtoken';
 import koa from 'koa';
-import koaMount from 'koa-mount';
-import authenticateRoutes from './authenticateRoutes';
+import koaRoute from 'koa-route';
+import methodFilter from '../lib/middlewares/methodFilter';
+import rateLimiter from '../lib/rateLimiter';
+import userRepositoryFactory from '../users/userModel';
 
 const app = koa();
 
-app.use(koaMount('/authenticate', authenticateRoutes));
+app.use(methodFilter(['POST']));
+app.use(koaRoute.post('/', rateLimiter(config.apps.api.security.rateLimitOptions)));
+
+app.use(koaRoute.post('/', function* login() {
+    const { email, password } = yield coBody(this);
+    const userRepository = userRepositoryFactory(this.client);
+    const user = yield userRepository.authenticate(email, password);
```

Comment:
Now, `userRepository.authenticate` return a `user` false if user is not authenticate. It's the controller which will throw an error (and set the status code to 401).